### PR TITLE
Await git.pullFrom and gitt.pushTo to ensure any failures display an …

### PR DESCRIPTION
**Pull request by chenhowa**
…error dialog to the user

### CODE REVIEW

**Comment by chenhowa** [comment]
Refers to  #41519.

**Comment by chenhowa** [comment]
This just adds two 'await' statements to ensure that if git.pushTo or git.pullFrom fail when using the Command Palette, the failure is not silent. Instead, the user is notified with an error dialog.

File: extensions/git/src/commands.ts [hunk 1]
```diff
@@ -1333,7 +1333,7 @@ export class CommandCenter {
 
 		const remoteCharCnt = remotePick.label.length;
 
-		repository.pullFrom(false, remotePick.label, branchPick.label.slice(remoteCharCnt + 1));
```
Comment (loridawn) [inline extensions/git/src/commands.ts]: #52116

Comment (PowerAccess) [reply extensions/git/src/commands.ts]: Hi @loridawn,
Is this related to implementing #52116 ("Extension Permission Management, Updating, Security & Categories") or did you mean to reference another issue?

**Comment by joaomoreno** [comment]
Great fix! Thanks! 🍻

**Pull request merged**